scss/_utilities.scss

```diff
@@ -135,3 +135,35 @@
 @include bg-variant('.bg-warning', $brand-warning);
 
 @include bg-variant('.bg-danger', $brand-danger);
+
+
+//
+// Utility functions
+//
+
+// Strips the unit from a single value entered.
+// E.g. `strip-unit(30px)` returns `30`
+
+@function strip-unit($value) {
+  @return $value / ($value * 0 + 1);
+}
+
+// Converts numeric value(s) to rem units, based
+// on the $font-size-root variable.
+// E.g. `rem-calc(14px 16px 0)` returns `0.875rem 1rem 0`
+
+@function rem-calc($values) {
+  $rem-values: ();
+
+  @each $value in $values {
+    @if ($value > 0) {
```

Comment:
Perhaps `@error` if `$value` is anything other than `rem`, `px`, and `0`; and pass-through `rem`?